hypersync-format/src/types/bloom_filter_wrapper.rs

```diff
@@ -11,7 +11,21 @@ use serde::{
 use crate::Data;
 
 #[derive(Debug, Clone)]
-pub struct FilterWrapper(sbbf_rs_safe::Filter);
+pub struct FilterWrapper(pub Filter);
+
+impl FilterWrapper {
+    pub fn new(bits_per_key: usize, num_keys: usize) -> Self {
+        FilterWrapper(Filter::new(bits_per_key, num_keys))
```

Comment:
I think you could try doing this via self, like Self{Filter::new(bits_per_key, num_keys)}